### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by mslukebo**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title: ''
+labels: enhancement, untriaged
+assignees: mslukebo
+
+---
+
+**Is your feature request an entirely new concept?**
+E.g. "I would like tables to support real-time data streaming."
+
+**Is your feature request related to an existing component?"
```
Comment (jmaxson-ms) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]: missing ending **

**Review by anramanath — APPROVED** [review]
Wondering, should we maybe add something in the glossary to link these two templates?

**Comment by mslukebo** [comment]
> Wondering, should we maybe add something in the glossary to link these two templates?

Nah, no need. They show up when someone starts creating a GitHub issue.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (mslukebo) [reply .github/ISSUE_TEMPLATE/feature_request.md:13]: Thanks!

**Review by anramanath — APPROVED** [review]

**Review by jmaxson-ms — APPROVED** [review]

**Pull request merged**